README.md

```diff
@@ -30,7 +30,8 @@ non-empty answer section. If so that is returned, otherwise the server at
 `8.8.8.8:53` will be checked. The order of checking is always in the order the
 `--fwd-to` parameters are specified.
 
-In reality struggledns will make a dns request for every `--fwd-to` at the same
-time, for every incoming request. It then checks responses in order. This way
-there's not much of a speed penalty, at the cost of more bandwidth and more hits
-to the remote servers.
+Each `--fwd-to` can be a group of comma-delimited addresses to query in
+parallel. Each group of `--fwd-to` addresses is queried in serial unless
+`--parallel` is specified, which causes **all** sent addresses from all groups
+to be queried in parallel. The response still respects the order of the
+addresses despite being sent in parallel.
```

Comment:
Can you add an example after this, like the one you sent me, to make it a bit more obvious how to use the grouping?